src/services/RecipeService.ts

```diff
@@ -27,4 +27,22 @@ export const getRecipeSearchResults = async (input: string) => {
     const response = request.json();
     console.log("RESPONSE: " + response);
     return response;
-}
\ No newline at end of file
+}
+
+export const getRecipeById = async (id: number) => {
+
+    const requestHeaders: HeadersInit = new Headers();
+    requestHeaders.set('Content-Type', 'application/json');
+
+    const baseUrl = 'https://api.spoonacular.com/recipes/' + id + '/information?apiKey=7d630837b6d0442e9dd50eb059343d02'
```

Comment:
u want to store api keys inside env variables - its fine here since its a free api but its a good habit to get into.

u can use the [dotenv](https://www.npmjs.com/package/dotenv) package to get ur env variables into ur frontend config somewhere.